lib/arena.rb

```diff
@@ -1,2 +1,50 @@
 class Arena
+
+# attr_accessor :name, :gladiators
+
+# * An arena has a name
+#   * The name should be capitalized
+# * An arena can have gladiators
+  def initialize (name)
+    @name = name
+    @gladiators = []
+  end
+
+  # * An arena has a name
+  #   * The name should be capitalized
+  def name
+    return @name.capitalize
```

Comment:
doesn't make a difference, but would prefer to see `.capitalize` called on the initial name passed in, on line 9